app.js

```diff
@@ -0,0 +1,58 @@
+import express from "express";
+import path from "path";
+import { db } from "./server/dbServer.js";
+
+export const getApp = () => {
+  const app = express();
+  app.use(express.urlencoded({ extended: true }));
+  app.use(express.json());
+  app.use(express.static("public"));
+
+  app.post("/todo", (req, res) => {
+    try {
+      const todo = db.addTodo(req.body.title);
+      return res.status(200).send(todo);
+    } catch (err) {
+      res.status(400).send({ error: err.message });
+    }
+  });
+
+  app.patch("/todo", (req, res) => {
+    try {
+      const todo = db.updateTodo(req.body.todo);
+      return res.status(200).send(todo);
+    } catch (err) {
+      res.status(400).send({ error: err.message });
+    }
+  });
+
+  app.delete("/todo/:id", (req, res) => {
+    try {
+      const todo = db.deleteTodo(req.params.id);
+      return res.status(200).send(todo);
+    } catch (err) {
+      res.status(400).send({ error: err.message });
+    }
+  });
+
+  app.delete("/todo-delete", (req, res) => {
```

Comment:
naming is bad. Pls rename to /todos
DELETE /todo/;id - you look at it and understand, that it deletes todo by id
DELETE /todo-delete - what does it delete?
I suggest
DELETE /todos - plural for todo. Means, that it will delete multiple